src/modules/CardsLogo.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <div class="cards">
+    <CardsList>
+      <slot />
+    </CardsList>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue'
+import CardsList from 'src/modules/CardsList.vue'
+
+export default defineComponent({
+  name: 'CardsLogo',
+  components: {
+    CardsList,
+  },
+})
+</script>
+
+<style scoped lang="scss">
+@import 'src/styles/widget.scss';
+
+@import 'src/styles/cards.scss';
```

Comment:
the div with class cards should be moved out of this file, then you can also remove this style import